modules/Microsoft.Compute/virtualMachines/deploy.bicep

```diff
@@ -380,7 +380,7 @@ module vm_nic '.bicep/nested_networkInterface.bicep' = [for (nicConfiguration, i
 resource vm 'Microsoft.Compute/virtualMachines@2021-07-01' = {
   name: name
   location: location
-  identity: identity
+  identity: any(identity)
```

Comment:
Is this intended?

Reply:
Ish. It was the working workaround for the most recent Bicep breaking bug that blocked us for some time. Can remove it now. Thanks for pointing it out

Reply:
removed